### PR TITLE
Union simple types

### PR DESCRIPTION
**Pull request by asaliev**
Addresses #9

### CODE REVIEW

File: docs/examples/basic.dto.neon [hunk 1]
```diff
@@ -7,6 +7,7 @@ Car:
 		attributes: 'string[]'
 		manufactured: '\Cake\I18n\FrozenDate'
 		owner: Owner
+		service: 'string[]|int|float'
```
Comment (dereuromark) [inline docs/examples/basic.dto.neon]: I would rather make this a test file in /tests/test_files
not the best example use case for the examples dtos IMO : )

File: docs/examples/basic.dto.xml [hunk 2]
```diff
@@ -12,6 +12,7 @@
 		<field name="attributes" type="string[]" />
 		<field name="manufactured" type="\Cake\I18n\FrozenDate" />
 		<field name="owner" type="Owner"/>
+		<field name="service" type="string[]|int|float"/>
```
Comment (dereuromark) [inline docs/examples/basic.dto.xml]: looks good
what about array with different types?

    string|int|float[]

how could this be done? whats the valid phpdoc for it?

File: docs/examples/basic.dto.neon (hunk 1 above)
Comment (asaliev) [reply docs/examples/basic.dto.neon]: Gotcha, will do

File: docs/examples/basic.dto.xml (hunk 2 above)
Comment (dereuromark) [reply docs/examples/basic.dto.xml]: OK

Comment (dereuromark) [reply docs/examples/basic.dto.xml]: I guess it's not official yet:
- https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#appendix-a-types
- https://youtrack.jetbrains.com/issue/WI-20193

**Comment by asaliev** [comment]
Anything pending that I've missed?

**Comment by dereuromark** [comment]
I didnt have a chance to review. Will do soon.

**Comment by dereuromark** [comment]
Did you try to render them via xml file? I dont see the validation xsd file adjusted.

**Pull request merged**